docs/UserGuide.adoc

```diff
@@ -129,9 +129,25 @@ This command allows you to view a recipe, the ingredients required, and the step
 
 - Format: `cookbook view recipe INDEX`
 - Example:
-* `cookbook view recipe 1` +
+* `cookbook view 1` +
     Views the ingredients and steps of recipe 1.
 
+==== Add a new recipe to the cookbook
```

Comment:
Maybe can add an optional description argument?

Reply:
Will add an optional argument for ingredients, steps and tags.